src/app/components/our-team/our-team.component.scss

```diff
@@ -87,22 +88,22 @@
       max-width: 235px;
       height: 270px;
       border-radius: 6px;
-      @include box-shadow(0px, 0px, 5px, 0px, $overlay-light);
+      @include box-shadow(0px, 0px, 10px, 5px, $overlay-light);  
       overflow: hidden;
       margin-bottom: 30px;
       margin-left: auto;
       margin-right: auto;
       transition: all 0.2s ease-in-out;
       &:hover {
-      	@include box-shadow(0px, 0px, 10px, 5px, $overlay-light);
+        @include box-shadow(0px, 0px, 5px, 0px, $overlay-light);
       }
   }
 
   .team-bg {
       position: absolute;
       top: -40px;
       left: -50px;
-      background: $blue-overlay-dark;
+      background: #252833;;
```

Comment:
Change the variable value `blue-overlay-dark`

Reply:
@Shekharrajak done